src/index.js

```diff
@@ -1,51 +1,75 @@
-import program from 'commander';
+import {Program} from './lib/program';
+import {Store} from './lib/config/store';
 import glob from 'glob';
 import logger from 'winston';
-
 import BundleCommand from './commands/bundle';
 import InitCommand from './commands/init';
 import NewCommand from './commands/new';
 
 class Aurelia {
+
   constructor() {
     this.commands = {};
+    this.aliases  = {};
```

Comment:
@ahmedshuhel 
Added this.aliases
This allows me to not have to loop through the commands for each argv.